InMobi/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.mopub.mobileads.inmobi">
```

Comment:
Check once, this should not be the same as the existing `CustomEventClass` that we ship

Reply:
This is a hard guideline by MoPub anyway if both the plugins have the same package name then there won't be any issues as we have used different class names than the previous plugin.

Reply:
No its not about the class names, it about the `BuildConfig` file which is generated by the Android Library that is created. `BuildConfig` classes should not have any conflict.